python-sdk/tutorials/automl-with-azureml/forecasting-many-models/auto-ml-forecasting-many-models.ipynb

```diff
@@ -366,6 +434,44 @@
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Construct your pipeline steps (Optional)\n",
```

Comment:
We may want to mention somewhere that we only support partitioned FileDataset and TabularDataset without partition when using such output as input.

Reply:
added a note.